docs/cloud/cloud-troubleshooting/issue-serverless-limitations.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Serverless limitations
+layout: default
+parent: Cloud troubleshooting
+grand_parent: Cloud
+nav_order: 4
+---
+{% include /serverless/serverless-preview-warning.md %}
+
+# Issue - Known limitations of serverless
+
+
+-   Some SQL functionality does not work:  
+    -   `GROUP BY` on [IDSET](/docs/sql-guide/data-types/data-type-idset) and [STRINGSET](/docs/sql-guide/data-types/data-type-stringset) data types  
+    -   `PERCENTILE`  
+    -   `HAVING`  
+    -   [IDSETQ](/docs/sql-guide/data-types/data-type-idsetq) and [STRINGSETQ](/docs/sql-guide/data-types/data-type-stringsetq) data types cannot be ingested
+    -   `TOP`
+    -   `LIMIT`   
+-   There is no backup nor restore (there is a snapshot store and write-logger that holds persistent data)
+{% include /serverless/serverless-worker-shape.md %}
```

Comment:
The items above this are FeatureBase features that are not supported.

However the worker-shape and worker limits are a part of the system and don't belong here.

My recommendation is to make this page JUST about functionality limitations within the boundary of what actually works with "standard" databases.

Reply:
No but it's a limitation, which we have an issue for cloud databases. I don't see this as different